### ISSUE TITLE
MD040 triggered on text starting with hard tab

### ISSUE THREAD

**Issue by ajaust** [markdownlint/markdownlint#397]
## Description

In some cases text preceded by a hard tab as in

hard-tab-file.md:
```markdown
# Header

	text after hard tab
```

rule MD040 is triggered. Rule MD040 is about missing language specifiers in fenced code blocks. 

This behavior was observed in #396. I wanted to open this PR to keep track of this bug.

### Environment

This problem **only** occurs if I activate all rules via a

.mdlrc:
```text
# Load style file (ruby script)
style '.mdl.rb'
```

and style file `.mdl.rb`:

```ruby
# Enable all rules by default
all

```

If I do **not** activate the all rules I get the expected behavior.

**MDL Version**
```bash
$ mdl --version
0.11.0
```

### Expected Behavior

Rule MD010 (Hard tabs) is triggered.

```bash
hard-tab-file.md:3: MD010 Hard tabs
```

### Actual Behavior

Rule MD010 and MD040 are triggered.

```bash
hard-tab-file.md:3: MD010 Hard tabs
hard-tab-file.md:3: MD040 Fenced code blocks should have a language specified
```

As far as I know, one cannot specify languages for code blocks that are created by 4 spaces or 1 hard tab. 

## Replication Case

Create a markdown file with the content from "Description" together with the configuration (`.mdlrc` and `.mdl.rb`) from the Section Environment.
